modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -1333,9 +1332,8 @@ public List<MBMessage> getGroupUserMessageBoardMessagesActivity(
 			MBMessageTable.INSTANCE
 		).where(
 			MBMessageTable.INSTANCE.modifiedDate.in(
-				DSLQueryFactoryUtil.select(
-					DSLFunctionFactoryUtil.max(
-						MBMessageTable.INSTANCE.modifiedDate)
+				DSLQueryFactoryUtil.selectDistinct(
+					MBMessageTable.INSTANCE.modifiedDate
```

Comment:
I haven't go trough very deeply, but why in one case you use the `selectDistinct` and in the other just `select`?

Reply:
You're right. I'll push-force the change..